### PR TITLE
feat(documentaion): hyperdrive styling

### PR DESCRIPTION
**Pull request by WouterBas**
**Description**
Hyperdrive themed styling

### CODE REVIEW

**Comment by IbenTesara** [comment]
Looks really good already, a great step in the right direction!

Do you think you can add some more color to the nav perhaps? Now it's a bit of a white snowfield, whilst we could add a pop of color.

Maybe the purple or the gradient from the homepage?
afbeelding
afbeelding

File: apps/docs/src/styles.scss [hunk 1]
```diff
@@ -1 +1,32 @@
 /* You can add global styles to this file, and also import other style files */
```
Comment (WHeirstrate) [inline apps/docs/src/styles.scss:1]: Small thing, but this line may be removed as well. 🙂

File: apps/docs/src/styles.scss [hunk 2]
```diff
@@ -1 +1,32 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@300..700&display=swap');
+
+h1,
+h2,
+h3 {
+	font-family: 'Space Grotesk', sans-serif !important;
+}
+
+:root {
+	--ng-doc-text-selection: rgba(187, 14, 253, 0.2);
```
[the remark this answers is not in the document]
Comment (WouterBas) [reply apps/docs/src/styles.scss:12]: Works on chrome, firefox and safari

**Comment by WouterBas** [comment]
new styling 
Screenshot 2024-12-26 at 14 42 22

**Review by WHeirstrate — APPROVED** [review]
Nice work! 💪

**Pull request merged**